terraform-v2/test/fmt/config.yml

```diff
@@ -0,0 +1,57 @@
+version: 2.1
```

Comment:
We should put in some terraform-v2 tests here

Reply:
My plan is to try and get the tests already in terraform-v2 working, then at some point take a look at filling in any gaps, there's probably some cmds that aren't covered. They look really useful though.